benchmark/cases/fake_connection_case.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/ev python3
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""
+Example performance test using benchmark framework.
+
+Test react speed on amount of incoming messages using normal agent operating.
+Messages are generated by fake connection.
+"""
+import time
+
+from benchmark.cases.helpers.dummy_handler import DummyHandler
+from benchmark.framework.aea_test_wrapper import AEATestWrapper
+from benchmark.framework.benchmark import BenchmarkControl
+from benchmark.framework.cli import TestCli
+
+from aea.configurations.base import SkillConfig
+
+
+def make_agency_conf(name: str = "dummy_agent", skills_num: int = 1) -> dict:
+    """Construct simple config for agency."""
+    return {
+        "name": "dummy_a",
+        "skills": [
+            {
+                "config": SkillConfig(name=f"sc{i}"),  # type: ignore
+                "handlers": {"dummy_handler": DummyHandler},
+            }
+            for i in range(skills_num)
+        ],
+    }
+
+
+def react_speed_in_loop(
+    benchmark: BenchmarkControl,
+    agents_num: int = 1,
+    skills_num: int = 1,
+    inbox_num: int = 5000,
+    agent_loop_timeout: float = 0.01,
+):
```

Comment:
missing params in docstring and `--> None` in signature

Reply:
done